src/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/no-unknown-property */
+/* eslint-disable react/prop-types */
+import cart from "../../images/cart.svg";
+import "./Cart.css";
+import CartItem from "./CartItem";
+
+const formatCurrency = new Intl.NumberFormat(undefined, {
+  style: "currency",
+  currency: "USD",
+});
+
+function Cart({
+  state,
+  isOpen,
+  handleClose,
+  removeItem,
+  updateItem,
+  clearCart,
+}) {
+  const purchaseHandler = () => {
+    clearCart();
+
+    setTimeout(() => {
+      alert("Thank you for your purchase!");
+    });
+  };
+
+  const cartTotalPrice = state.reduce(
+    (acc, { price, quantity }) => price * quantity + acc,
+    0
+  );
+
+  if (!isOpen) return null;
+
+  if (state.length === 0) {
+    return (
+      <div className="cart__container">
+        <div className="cart__buttons">
+          <button className="purchase__button" disabled>
+            Purchase
+          </button>
+          <button className="close__button" onClick={handleClose}>
+            X
+          </button>
+        </div>
+        <img src={cart} alt="cart" width="40" />
```

Comment:
BTW we can use svg as React components:
https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs